pkg/processor/procbuiltin/func_wrapper.go

```diff
@@ -0,0 +1,60 @@
+// Copyright © 2023 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package procbuiltin
+
+import (
+	"context"
+
+	"github.com/conduitio/conduit/pkg/foundation/log"
+	"github.com/conduitio/conduit/pkg/inspector"
+	"github.com/conduitio/conduit/pkg/record"
+	"github.com/rs/zerolog"
+)
+
+// FuncWrapper is an adapter allowing use of a function as an Interface.
+type FuncWrapper struct {
```

Comment:
Previously `InterfaceFunc`. I've moved it to the `procbuiltin` package, because this is virtually the only place where it's used (plus a couple of tests).